handlers/age.go

```diff
@@ -170,18 +195,27 @@ func (f *Filter) addAgeRange(filterID string, req *http.Request) error {
 			isInRange = false
 		}
 	}
-
-	return f.FilterClient.AddDimensionValues(req.Context(), filterID, "age", options)
+	return f.FilterClient.AddDimensionValues(ctx, userAccessToken, serviceAuthToken, collectionID, filterID, dimensionName, options)
 }
 
 func (f *Filter) Age(w http.ResponseWriter, req *http.Request) {
 	vars := mux.Vars(req)
 	filterID := vars["filterID"]
 	ctx := req.Context()
+	dimensionName := "age"
 
-	req = forwardFlorenceTokenIfRequired(req)
+	collectionID := getCollectionIDFromContext(ctx)
+	userAccessToken, err := headers.GetUserAuthToken(req)
+	if !headers.IsNotFound(err) {
+		log.Error(err, nil)
+	}
+
+	serviceAuthToken, err := headers.GetServiceAuthToken(req)
+	if err != nil{
```

Comment:
use `!headers.IsNotFound(err)` here too.

Reply:
`serviceAuthToken` removed.
So from what I understand now is that the controller will never directly fetch private content so serviceAuthToken isn't needed. However, it will fetch private content if used via Florence, if so a `userAuthToken` would be present instead. As per James' comment have removed serviceAuthToken and just sending the argument as an empty string to the APIs.